models/src/main/protobuf/CasperMessage.proto

```diff
@@ -107,23 +107,6 @@ message BlockHashMessageProto {
   bytes  blockCreator = 2;
 }
 
-message BlockMetadataInternal {
```

Comment:
Would be nice at some point to leave in CasperMessage only really messages related to Casper, and move everything else to another file.

Reply:
I tried that, but BlockMetadata has dependency on types from CasperMessage. If multiple generators are specified they will include all dependency and generate duplicates.
https://github.com/rchain/rchain/blob/069b4093a47d9545528224b4e025817896ce27c0/build.sbt#L243-L246

We should not have internal types defined in the same way as protocol or external types.